memote-report-app/src/app/tauchart-history/tauchart-history.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, OnInit, ViewEncapsulation, Input, AfterViewInit } from '@angular/core';
+import { ReportDataService } from '../report-data.service';
+import { Chart, api } from 'taucharts';
+import 'taucharts/dist/plugins/tooltip';
+import 'taucharts/dist/plugins/legend';
+import 'taucharts/dist/plugins/export-to';
+
+@Component({
+  selector: 'app-tauchart-history',
+  template: '<div id="tau-history-{{css_tag}}" style="height: 50vh; width: 40vw"> </div>',
+  encapsulation: ViewEncapsulation.None
+})
+export class TauChartHistoryComponent implements OnInit, AfterViewInit {
+  @Input() testObject: any;
+  @Input() testId: string;
+  format_type: string;
+  chart: any;
+  historyData: Object[];
+  css_tag = 'total';
```

Comment:
I'd maybe call this a `unique_id` or similar instead - looks to me like it's not really used for css, just as a unique html id attribute.

Reply:
Good call! It is indeed.